### PR TITLE
Add namespaced filters, progress information and continue with exclude filters display on error or user interrupt

### PR DESCRIPTION
**Pull request by Skarlso**

### CODE REVIEW

File: pkg/collections.go [hunk 1]
```diff
@@ -25,16 +25,20 @@ package pkg
...
+
 	"github.com/arangodb/go-driver"
 	"github.com/rs/zerolog/log"
 	"golang.org/x/sync/errgroup"
 	"golang.org/x/sync/semaphore"
 )
 
 // copyCollections copies all collections for a database.
-func (c *copier) copyCollections(ctx context.Context, db driver.Database) error {
+func (c *copier) copyCollections(ctx context.Context, db driver.Database, doneCollections chan string) error {
```
Comment (ewoutp) [inline pkg/collections.go]: Prefer to strong type the channel.
Something like 

type databaseAndCollection struct {
DatabaseName string 
CollectionName string
}

doneCollection chan databaseAndCollection

Comment (Skarlso) [reply pkg/collections.go]: I was actually leaning that way. I'll do that!

File: pkg/copier.go [hunk 2]
```diff
@@ -107,12 +109,24 @@ type copier struct {
 	graphExclude      map[string]struct{}
 }
 
+// databaseAndCollections is the type of the done channel for copying data across.
+type databaseAndCollections = struct {
+	databaseName   string
+	collectionName string
```
Comment (ewoutp) [inline pkg/copier.go:118]: Please add comment that this field can be empty

**Review by ewoutp — APPROVED** [review]
1 comment, then LGTM

**Pull request merged**